src/ui/simulator/toolbox/components/datagrid/renderer/ts-management-clusters-as-renewables.cpp

```diff
@@ -65,461 +51,12 @@ wxString TSmanagementRenewableCluster::columnCaption(int colIndx) const
                                              wxT("   Thermal   "),
```

Comment:
The captions should be retrieved from de columns, not fixed one for all in the TS manager